guides/doc-Monitoring_Project/topics/proc_installing-pcp-packages.adoc

```diff
@@ -0,0 +1,35 @@
+[id='installing-pcp-packages_{context}']
+= Installing PCP Packages
+
+This procedure describes how to install the PCP packages.
+
+.Prerequisites
+
+* Ensure you have a minimum of 20 GB space available in the `/var/log/pcp` directory.
++
+The default PCP data retention policy is to retain only that data collected during the past 14 days. Data storage per day is estimated to use usually between 100 MB and 500 MB of disk space, but may use up to several gigabytes.
+
+.Procedure
+
+. Install the PCP packages:
++
+[options="nowrap", subs="verbatim,quotes,attributes"]
+----
+ifndef::foreman-deb[]
+# {package-install-project} pcp \
+  pcp-pmda-apache \
+  pcp-pmda-openmetrics \
+  pcp-pmda-postgresql \
+  pcp-pmda-redis \
+  pcp-system-tools
+endif::[]
+ifdef::foreman-deb[]
+# {package-install-project} pcp
+endif::[]
+----
+
+. Enable and start the Performance Metrics Collector daemon, and the Performance Metrics Logger daemon:
++
+----
+# systemctl enable --now pmcd pmlogger
```

Comment:
Does PCP handle services being down elegantly? In the case of a restart or an upgrade.

Reply:
If the services it is monitoring are down, I assume it handles that gracefully?

Reply:
As far as I have seen, yeah.